### DISCUSSION TITLE
How to run the command line displayed in the Console window during the OMNet++simulation process through commands in mingwenv.cmd

### DISCUSSION THREAD

**Discussion by mirror1314** [omnetpp/omnetpp#1008] [Q&A] [answered]
Hello,
The problem is as shown in the title. When I input the command line in Console in mingwenv.cmd, the second line reported an error. How can I modify the command to execute simulation in cmd?
/d/software/omnetpp-5.4.1/samples/CoRE4INET/simulations/ADAS_and_ENT/star/AS_Qbv_Qci_Qbu$ ../../../../../../bin/opp_run.exe -m -u Qtenv -c General -n ../../../../examples;../../..;../../../../src;../../../../../inet/src;../../../../../inet/examples;../../../../../inet/tutorials;../../../../../inet/showcases --image-path=../../../../images;../../../../../inet/images -l ../../../../src/CoRE4INET -l ../../../../../inet/src/INET AS_Qbv_Qci_Qbu.ini
OMNeT++ Discrete Event Simulation  (C) 1992-2018 Andras Varga, OpenSim Ltd.
Version: 5.4.1, build: 180629-5e28390, edition: Academic Public License -- NOT FOR COMMERCIAL USE
See the license for distribution terms and warranty disclaimer

<!> Error: Cannot open ini file 'omnetpp.ini'

End.
bash: ../../..: Is a directory
bash: ../../../../src: Is a directory
bash: ../../../../../inet/src: Is a directory
bash: ../../../../../inet/examples: Is a directory
bash: ../../../../../inet/tutorials: Is a directory
bash: ../../../../../inet/showcases: Is a directory
bash: ../../../../../inet/images: Is a directory

Thanks!

**Comment by rhornig** [comment, marked as answer]
In the ned path (`-n`) all semicolon (`;`) must be changed to colon (`:`). (or you should quote the full ned path (i.e. put into "").

**Reply by mirror1314** [reply]
Thanks!
But I still encounter a problem now, when I modify the command according to your suggestion and report an error.
/d/software/omnetpp-5.4.1/samples/CoRE4INET/simulations/ADAS_and_ENT/star/AS_Qbv_Qci_Qbu$ ../../../../../../bin/opp_run.exe AS_Qbv_Qci_Qbu.ini -m -u Qtenv -n ../../../../examples:../../..:../../../../src:../../../../../inet/src:../../../../../inet/examples:../../../../../inet/tutorials:../../../../../inet/showcases: --image-path=../../../../images:../../../../../inet/images;-l ../../../../src/CoRE4INET -l ../../../../../inet/src/INET
OMNeT++ Discrete Event Simulation  (C) 1992-2018 Andras Varga, OpenSim Ltd.
Version: 5.4.1, build: 180629-5e28390, edition: Academic Public License -- NOT FOR COMMERCIAL USE
See the license for distribution terms and warranty disclaimer

Setting up Qtenv...

Loading NED files from ..\..\..\..\examples:  100
Loading NED files from ..\..\..:  28
Loading NED files from ..\..\..\..\src:  208
Loading NED files from ..\..\..\..\..\inet\src:  666
Loading NED files from ..\..\..\..\..\inet\examples:  174
Loading NED files from ..\..\..\..\..\inet\tutorials:  19
Loading NED files from ..\..\..\..\..\inet\showcases:  33

libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
Loading images from 'D:\software\omnetpp-5.4.1\samples\CoRE4INET\images': *: 0  background/*: 3  buffer/*: 12  device/*: 2
Loading images from 'D:\software\omnetpp-5.4.1\samples\inet\images': *: 0  background/*: 1  misc/*: 22
Loading images from 'D:\software\omnetpp-5.4.1\images': *: 0  abstract/*: 90  background/*: 4  block/*: 320  device/*: 195  logo/*: 1  maps/*: 9  misc/*: 70  msg/*: 55  old/*: 111  status/*: 28


End.
bash: -l: Command not found

**Reply by rhornig** [reply]
You still clearly have a semicolon in that command line (at the end of the ned-path).

**Reply by mirror1314** [reply]
Sorry.Because in my countless experiments, not adding this semicolon would cause one of the folders to report an error when loading the DLL file, causing me to have to add a semicolon. If this semicolon is placed at the end of - l, it will also report the same error, such as before - l.
/d/software/omnetpp-5.4.1/samples/CoRE4INET/simulations/ADAS_and_ENT/star/AS_Qbv_Qci_Qbu$ ../../../../../../bin/opp_run.exe AS_Qbv_Qci_Qbu.ini -m -u Qtenv -n ../../../../examples:../../..:../../../../src:../../../../../inet/src:../../../../../inet/examples:../../../../../inet/tutorials:../../../../../inet/showcases: --image-path=../../../../images:../../../../../inet/image -l ../../../../src/CoRE4INET -l ../../../../../inet/src/INET
OMNeT++ Discrete Event Simulation  (C) 1992-2018 Andras Varga, OpenSim Ltd.
Version: 5.4.1, build: 180629-5e28390, edition: Academic Public License -- NOT FOR COMMERCIAL USE
See the license for distribution terms and warranty disclaimer


<!> Error: Cannot load library '../../../../src//libCoRE4INET.dll': ▒Ҳ▒▒▒ָ▒▒▒▒ģ▒▒

End.
